includes/Includes_Neopets_Auctions/main.user.js

```diff
@@ -0,0 +1,95 @@
+// ==UserScript==
+// @name        Neopets : Includes : Auctions
+// @namespace   github.com
+// @include     nowhere
+// @exclude     *
+// @version     1
+// @grant       GM_xmlhttpRequest
+// @grant       GM_getResourceText
+// @require     http://code.jquery.com/jquery-latest.js
+// @require     https://github.com/knadh/localStorageDB/raw/master/localstoragedb.min.js
+// @require     https://github.com/w35l3y/userscripts/raw/master/includes/Includes_XPath/63808.user.js
+// @require     https://github.com/w35l3y/userscripts/raw/master/includes/Includes_HttpRequest/56489.user.js
+// @require     https://github.com/w35l3y/userscripts/raw/master/includes/Includes_Neopets_%5BBETA%5D/main.user.js?2387
+// ==/UserScript==
+
+// auctions.phtml?type=placebid
+// { _ref_ck : neo.ck, amount : 245000, auction_id : auctionid }
+
+var Auctions = function(page){
+    var _post = function(data, cb){
+		page.request({
+			method	: "post",
+			action	: "http://www.neopets.com/auctions.phtml",
+			referer	: referer,
+			data	: data,
+			delay	: true,
+			callback: cb
+		});
+    },
+    _get = function(data,cb){
+        page.request({
+           method   : "get",
+           action   : "http://www.neopets.com/auctions.phtml",
+           data     : data,
+           delay    : true,
+           callback : cb
+        });
+    };
+    
+    this.info = function(obj){
```

Comment:
I changed `info` to `bids`.
The result also is different to follow the pattern: `{error, errmsg, body, response}`
The structure of `response` is: `{id, item: {name, image, description}, owner, nf_only, timeleft, increment, currentprice, list: [{bidder, amount, when}]}` (may be seen at `Auctions.parse("bids")`)